easycv/toolkit/blade/cv_blade_utils.py

```diff
@@ -0,0 +1,248 @@
+import argparse
+import itertools
+import os
+import time
+import timeit
+import logging
+from contextlib import contextmanager
+
+import numpy as np
+import pandas as pd
+import torch
+import torchvision
+
+os.environ['DISC_ENABLE_STITCH'] = os.environ.get('DISC_ENABLE_STITCH', 'true')
+os.environ['DISC_EXPERIMENTAL_SPECULATION_TLP_ENHANCE'] = os.environ.get('DISC_EXPERIMENTAL_SPECULATION_TLP_ENHANCE', 'true')
+
+try:
+    import torch_blade
+    import torch_blade.tensorrt
+    from torch_blade import optimize
+    import ctypes
+    _cudart = ctypes.CDLL('libcudart.so')
```

Comment:
Nit: not needed here.

Reply:
will cause a import error even we dont use it.